backend/api/btb/api/constants.py

```diff
@@ -0,0 +1,8 @@
+from enum import IntEnum
+
+# also update frontend/constants/inputLengths.ts
```

Comment:
Couldn't we export this to FE?

Reply:
Yes I would like this too but I don't know a nice way. Extending the GraphQL Api for this seems overkill.